tests/blackbox/src/common/metrics.py

```diff
@@ -0,0 +1,66 @@
+class Metrics:
+    """Singleton class to store metrics."""
+
+    __instance = None
```

Comment:
Q: Why does this have a double-underscore?

Reply:
https://dbader.org/blog/meaning-of-underscores-in-python